modules/ossm-servicemesh-overview.adoc

```diff
@@ -1,21 +1,16 @@
 ////
 Module included in the following assemblies:
-* service_mesh/v1x/ossm-architecture.adoc
-* service_mesh/v2x/ossm-architecture.adoc
-* service_mesh/v1x/servicemesh-release-notes.adoc
-* service_mesh/v2x/servicemesh-release-notes.adoc
+* service_mesh/v2x/ossm-about.adoc
 ////
 
 [id="ossm-servicemesh-overview_{context}"]
-= {ProductName} overview
+= Introduction to {ProductName}
 
-{ProductName} is a platform that provides behavioral insight and operational control over the service mesh, providing a uniform way to connect, secure, and monitor microservice applications.
+{ProductName} is designed to address a variety of problems in a microservice architecture by creating a centralized point of control in an application. It adds a transparent layer on existing distributed applications without requiring any changes to the application code.
```

Comment:
Small nit pick:

I would change passive voice to active voice. So, you could instead say something like, 

`{ProductName} addresses a variety of problems in a microservice...It adds a transparent layer...`

Since your second sentence is active voice, I think it reads better for the first sentence to be active as well.